src/main/java/org/kohsuke/github/GHEventPayload.java

```diff
@@ -454,19 +411,8 @@ void wrapUp(GitHub root) {
      * @see <a href="https://docs.github.com/en/rest/reference/pulls#reviews">Pull Request Reviews</a>
      */
     public static class PullRequestReview extends GHEventPayload {
-        private String action;
         private GHPullRequestReview review;
-        private GHPullRequest pull_request;
```

Comment:
note: snake_case replaced with camelCase, mapper is set up to force snake_case, so it was redundant